ui/app/helpers/utils/i18n-helper.js

```diff
@@ -55,10 +55,12 @@ export const getMessage = (localeCode, localeMessages, key, substitutions) => {
         return part
       }
       const substituteIndex = Number(subMatch[1]) - 1
-      if (substitutions[substituteIndex]) {
-        return substitutions[substituteIndex]
+      if (substitutions[substituteIndex] == null) {
+        const error = new Error(`Insufficient number of substitutions for message: '${phrase}'`)
+        log.error(error)
+        Sentry.captureException(error)
       }
```

Comment:
Should this branch return or throw? Or is returning `null` below fine?

Reply:
Well, throwing here was the entire problem this PR is solving. I'm not sure what you're asking exactly.

Reply:
Oh, if you mean to ask how `null` differs from `undefined`, they're exactly the same for the string substitution case (both are rendered as empty strings). There's a test case for this.

If one of the other substitutions is detected as a React substitution (i.e. a non-null Object or function), then both `null` and `undefined` are rendered as empty `span` elements. This one isn't in a unit test.

Reply:
I meant more that we could maybe `return null` here explicitly instead of allowing the return below? It seems weird to have what looks like error handling that doesn't terminate the function. But this is fine for now.